src/status_im/ui/screens/chat/bottom_sheets/context_drawer.cljs

```diff
@@ -0,0 +1,84 @@
+(ns status-im.ui.screens.chat.bottom-sheets.context-drawer
+  (:require [quo.react-native :as rn]
+            [status-im.ui.components.react :as react]
+            [re-frame.core :as re-frame]
+            [status-im.ui.screens.wallet.components.views :as components]
+            [status-im.constants :as constants]
+            [quo2.components.buttons.button :as quo2.button]
+            [quo2.foundations.colors :as quo2.colors]
+            [quo2.components.list-item :as quo2.list-item]))
+
+(defn message-options [actions own-reactions send-emoji]
+  (fn []
+    (let [main-actions (filter #(= (:type %) :main) actions)
+          danger-actions (filter #(= (:type %) :danger) actions)
+          admin-actions (filter #(= (:type %) :admin) actions)]
+      [react/view {:flex 1}
+       [react/view {:style {:width "100%"
+                            :flex-direction :row
+                            :justify-content :space-between
+                            :padding-horizontal 30
+                            :padding-top 5
+                            :padding-bottom 15}}
+        (doall
+         (for [[id resource] constants/reactions
+               :let          [active (own-reactions id)]]
+           ^{:key id}
+           [quo2.button/button (merge
+                                {:width               40
+                                 :size                40
+                                 :type                :grey
+                                 :accessibility-label :reply-cancel-button
+                                 :on-press            #(do
+                                                         (send-emoji id)
+                                                         (re-frame/dispatch [:bottom-sheet/hide]))}
+                                (when active {:style {:background-color quo2.colors/neutral-70}}))
+            [rn/image {:source resource
+                       :style  {:height 20
+                                :width  20}}]]))]
+       [rn/view {:style {:padding-horizontal 8}}
+        (for [action main-actions]
+          ^{:key (:id action)}
+          (let [on-press (:on-press action)]
+            [quo2.list-item/list-item
```

Comment:
lets discuss this , but im not sure we need to use such complex heavy component for a such simple item , probably its better to have simple component like menu-item

Reply:
not against it, maybe we can keep it and simplify the component?

Reply:
tbh for me, it looks simpler to create a new component because it's really simple, don't see how this one could be simplified

Reply:
Will work on it then.

Reply:
Removed quo2 list-item in favor of a simpler component menu-item. Can you re-review it please?